src/Lib/Items/CanvasImage.vala

```diff
@@ -85,17 +85,16 @@ public class Akira.Lib.Items.CanvasImage : Goo.CanvasImage, Akira.Lib.Items.Canv
                 pixbuf = original_pixbuf;
 
                 // Define the item's size based on the images size.
-                size.width = original_pixbuf.width;
-                size.height = original_pixbuf.height;
+                size.width = original_pixbuf.width/2;
+                size.height = original_pixbuf.height/2;
```

Comment:
Why are you doing this?
You shouldn't randomly divide the size as you don't know which image size the user might add.